homeassistant/util/redact.py

```diff
@@ -0,0 +1,71 @@
+"""Helpers to redact Google Assistant data when logging."""
```

Comment:
If this was just google assistant related, it should live in that integration. I think its more generic?

Reply:
It was in that integration to start with, but was moved, so that is now a stale docstring 👍